haya-hayaodeh/Homework-1/fizzbuzz/app.js

```diff
@@ -25,3 +25,19 @@
 // ##### Step 4:
 // Add an additional `else if` clause that logs the string `"fizzbuzz"` if the value being iterated over is divisible by both `3` and `5`.
 
+for (let i = 1; i < 101; i++) {
```

Comment:
This `for` statement does the job, but I think it would be clearer to say `i <= 100` rather than `i < 101` -- even though they both get you to the same result, the first one helps other developers understand more quickly what your code does.